src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -9,15 +9,24 @@ import SummaryChartGroup from '../components/charts/SummaryChartGroup/.';
 import PercentageStackedBarChart from '../components/charts/PercentageStackedBarChart/.';
 import DataSummaryCardGroup from '../components/cards/DataSummaryCardGroup/.';
 import { getCharts } from '../components/charts/helper';
-import { downloadManifest, downloadData, getManifestEntryCount } from './actionHelper';
+import { downloadManifest, downloadData, getManifestEntryCount, exportToWorkspace } from './actionHelper';
 import { calculateDropdownButtonConfigs, humanizeNumber } from './utils';
 import { exportAllSelectedDataToCloud } from './custom/bdbag';
+import { workspaceUrl } from '../localconf';
 
 class DataExplorerVisualizations extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       manifestEntryCount: 0,
+      exportInProgress: false,
+      exportedToWorkspace: false,
```

Comment:
I think you can combine this var and the `exportErrorStatus` variable since they both are indicated success?